<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>1</y>
      <width>356</width>
      <height>94</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="lista_comboBox"/>
         </item>
         <item>
          <widget class="QLabel" name="mese_lbl">
           <property name="text">
            <string>Mese</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="giorni_spinbox">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::PlusMinus</enum>
           </property>
           <property name="keyboardTracking">
            <bool>true</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>31</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="giorno_lbl">
           <property name="text">
            <string>Giorno</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mesi_spinbox">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::PlusMinus</enum>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>12</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="importo_lbl">
           <property name="text">
            <string>Importo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="importo_text"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="continua_button">
         <property name="text">
          <string>Continua</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="calcola_button">
         <property name="text">
          <string>Calcola</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modifica_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Modifica</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuovo"/>
  </widget>
  <action name="actionNuovo">
   <property name="text">
    <string>Nuovo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
